src/SFA.DAS.EmployerFinance/Configuration/GoogleAnalyticsConfigurationFactory.cs

```diff
@@ -0,0 +1,32 @@
+using SFA.DAS.AutoConfiguration;
+
+namespace SFA.DAS.EmployerFinance.Configuration
+{
+    public class GoogleAnalyticsConfigurationFactory : IGoogleAnalyticsConfigurationFactory
```

Comment:
This will need registering in the container.

Reply:
done